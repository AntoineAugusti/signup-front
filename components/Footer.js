import React from 'react';

const Footer = () => (
  <footer className="footer">
      <div className="container">
        <div className="footer__logo">
          <img src="/static/images/etalab.svg" alt="Logo Etalab" />
          <ul className="footer__social">
            <li><a href="https://twitter.com/etalab" title="Twitter"><img src="/static/images/social/twitter.svg" alt="Twitter" className="icon icon-twitter" /></a></li>
            <li><a href="https://github.com/etalab" title="Github"><img src="/static/images/social/github.svg" alt="Github" className="icon icon-github" /></a></li>
            <li><a href="https://www.facebook.com/etalab" title="Facebook"><img src="/static/images/social/facebook.svg" alt="Facebook" className="icon icon-facebook" /></a></li>
            <li><a href="mailto:contact@particulier.api.gouv.fr" title="Nous écrire un mail"><img src="/static/images/social/email.svg" alt="Email" className="icon icon-mail" /></a></li>
          </ul>
        </div>
        <ul className="footer__links">
          <li><h2>template.data.gouv.fr</h2></li>
          <li><a href="https://www.data.gouv.fr/reference">Données de référence</a></li>
          <li><a href="https://www.data.gouv.fr/terms">Conditions générales d'utilisation</a></li>
          <li><a href="https://www.data.gouv.fr/api">API</a></li>
        </ul>
        <ul className="footer__links"></ul>
      </div>
  </footer>
);

export default Footer;
