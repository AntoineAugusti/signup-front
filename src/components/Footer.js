import React from 'react'

const Footer = () => (
  <footer className='footer'>
    <div className='container'>
      <div className='footer__logo'>
        <a href='https://api.gouv.fr/'>
          <img src='/images/logo-api.gouv.fr-white.png' alt='Logo api.gouv.fr' />
        </a>
        <ul className='footer__social'>
          <li><a href='https://twitter.com/BetaGouv' title='Twitter'><img src='/images/social/twitter.svg' alt='Twitter' className='icon icon-twitter' /></a></li>
          <li><a href='https://github.com/betagouv' title='Github'><img src='/images/social/github.svg' alt='Github' className='icon icon-github' /></a></li>
          <li><a href='mailto:contact@particulier.api.gouv.fr' title='Nous Ã©crire un mail'><img src='/images/social/email.svg' alt='Email' className='icon icon-mail' /></a></li>
        </ul>
      </div>
      <ul className='footer__links'>
        <li><h2>particulier.api.gouv.fr</h2></li>
        <li><a href='https://status.particulier.api.gouv.fr'>Status des API</a></li>
      </ul>
    </div>
  </footer>
)

export default Footer
