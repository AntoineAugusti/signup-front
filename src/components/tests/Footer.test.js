import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import Footer from '../Footer';

describe('components | Footer', () => {
  describe('render', () => {
    const renderer = ReactTestRenderer.create(<Footer />);
    it('should be defined', () => {
      expect(renderer).toBeDefined();
    });
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot();
    });
  });
});
