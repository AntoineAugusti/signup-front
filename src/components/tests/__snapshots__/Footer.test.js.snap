// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | Footer render should match snapshot 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="footer__logo"
    >
      <a
        href="https://api.gouv.fr/"
      >
        <img
          alt="Logo api.gouv.fr"
          src="/images/logo-api.gouv.fr-white.png"
        />
      </a>
      <ul
        className="footer__social"
      >
        <li>
          <a
            href="https://twitter.com/BetaGouv"
            title="Twitter"
          >
            <img
              alt="Twitter"
              className="icon icon-twitter"
              src="/images/social/twitter.svg"
            />
          </a>
        </li>
        <li>
          <a
            href="https://github.com/betagouv"
            title="Github"
          >
            <img
              alt="Github"
              className="icon icon-github"
              src="/images/social/github.svg"
            />
          </a>
        </li>
        <li>
          <a
            href="mailto:contact@particulier.api.gouv.fr"
            title="Nous Ã©crire un mail"
          >
            <img
              alt="Email"
              className="icon icon-mail"
              src="/images/social/email.svg"
            />
          </a>
        </li>
      </ul>
    </div>
    <ul
      className="footer__links"
    >
      <li>
        <h2>
          particulier.api.gouv.fr
        </h2>
      </li>
      <li>
        <a
          href="https://status.particulier.api.gouv.fr"
        >
          Status des API
        </a>
      </li>
    </ul>
  </div>
</footer>
`;
